engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/runtime/JobDefinitionSuspensionStateDto.ftl

```diff
@@ -0,0 +1,83 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/job-definition/put-activate-suspend-by-id/index.html -->
+<#macro dto_macro docsUrl="">
+
+<#assign noteMutualExclusive = "Note that only one of `processDefinitionId`, `processDefinitionKey` or a job definition
+                                id (as path parameter) is allowed for a request." >
+<#assign noteProcessDefinitionKey = "Note that this parameter will only be considered 
+                                     in combination with `processDefinitionKey`." >
+
+<@lib.dto>
+    
+    <@lib.property
+        name = "processDefinitionId"
+        type = "string"
+        desc = "The process definition id of the job definitions to activate or suspend.
+        
+                ${noteMutualExclusive}"
+    />
+    
+    <@lib.property
+        name = "processDefinitionKey"
+        type = "string"
+        desc = "The process definition key of the job definitions to activate or suspend.
+        
+                ${noteMutualExclusive}"
+    />
+
+    
+    <@lib.property
+        name = "processDefinitionTenantId"
+        type = "string"
+        desc = "Only activate or suspend job definitions of a process definition which belongs to a
+                tenant with the given id.
+                
+                ${noteProcessDefinitionKey}"
+    />
+
+    
+    <@lib.property
+        name = "processDefinitionWithoutTenantId"
+        type = "string"
+        desc = "Only activate or suspend job definitions of a process definition which belongs to
+                no tenant. Value may only be `true`, as `false` is the default
+                behavior.
+                
+                ${noteProcessDefinitionKey}"
+    />
+
+    
+    <@lib.property
+        name = "suspended"
+        type = "boolean"
+        desc = "A `Boolean` value which indicates whether to activate or suspend the referenced job
+                definitions. When the value is set to `true`, the job
+                definitions will be suspended and when the value is set to `false`,
+                they will be activated."
+    />
+
+    
+    <@lib.property
+        name = "includeJobs"
+        type = "boolean"
+        desc = "A `Boolean` value which indicates whether to activate or suspend also all jobs of
+                the referenced job definitions. When the value is set to `true`, all jobs
+                of the provided job definitions will be activated or suspended and
+                when the value is set to `false`, the suspension state of all jobs
+                of the provided job definitions will not be updated."
+    />
+
+    
+    <@lib.property
+        name = "executionDate"
+        type = "string"
+        format = "date-time"
```

Comment:
This property is just a `String` in the `JobDefinitionSuspensionStateDto` Java class. It's parsed into a `Date` object when it's applied to the `UpdateJobDefinitionSuspensionStateBuilder` instance.

I don't think it will make a difference if it has the `format = "date-time"` here. We can do the following:
1. Leave it like this :)
2. Remove it, so that it's the same as the Java code;
3. Refactor the Java property to a Date type.

Reply:
I just got carried away by the description of the property, I will remove the `format` 👍 😃